sympy/stats/tests/test_discrete_rv.py

```diff
@@ -56,8 +56,8 @@ def test_Poisson():
         import numpy as np
     except ImportError:
         skip("numpy not installed")
-    y = Poisson('y', np.float64(4.72544290380919e-11))
-    assert E(y) == 4.72544290380919e-11
+    y = Poisson('y', np.float64(4.725442903809197e-11))
+    assert E(y) == 4.725442903809197e-11
```

Comment:
Why does it need to be this exact value?

Reply:
If the purpose is to verify that the expected value of Poisson(λ) is λ, then any value would suffice. However, since the issue number is referenced in the comment, readers of the code would expect this test to be related to that specific issue. While `4.72544290380919e-11` does not raise an error and `4.725442903809197e-11` does, I thought the latter should be tested.

On further reflection, though, even the former does not produce the correct result despite not raising an error. It might be better to close this PR.

Reply:
I guess we can just test both values.